src/components/TeacherComponents/ListItemHeader/ListItemHeader.tsx

```diff
@@ -22,13 +23,15 @@ interface CommonProps {
   hideTopHeading?: boolean;
   primaryTargetProps: PrimaryTargetProps;
   page: "Unit" | "Lesson";
-  onClick?: () => void;
+  onClick: () => void;
 }
 
-type SpecialistListItemProps = IndividualSpecialistUnit &
+type SpecialistListItemProps = (IndividualSpecialistUnit | SpecialistLesson) &
   CommonProps & {
-    title: LessonListItemProps["lessonTitle"] | UnitListItemProps["title"];
-    slug: LessonListItemProps["lessonSlug"] | UnitListItemProps["slug"];
+    title: SpecialistLesson["lessonTitle"] | IndividualSpecialistUnit["title"];
+    slug: SpecialistLesson["lessonSlug"] | IndividualSpecialistUnit["slug"];
```

Comment:
I know you didn't add this here, but these are just strings right? I'm not sure the extra typing is helpful 😅

Reply:
ahaha yep, changed✅